code/__defines/misc.dm

```diff
@@ -160,7 +160,7 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 #define MAX_PAPER_MESSAGE_LEN 3072
 #define MAX_BOOK_MESSAGE_LEN  9216
 #define MAX_LNAME_LEN         64
-#define MAX_NAME_LEN          26
+#define MAX_NAME_LEN          63
```

Comment:
Issue I see with this is that at least 58(presumably 74) things in code use this define. Mainly objects like bodybag lables, revolver, pistols, chemistry lables.

63 character ~= 12 words if we assume average word is 5 characters. That is long ass name for bodybag label

My suggestion is to seperate Mob's max name define and Object's max name in two defines.  One way would be to create new define like `MAX_NAME_MOB_LEN`, and also rename `MAX_LNAME_LEN` to be similar.

Reply:
I am fully aware of how many objects use this value. Why do you care if a bodybag label name is long? The intent of this PR is _precisely_ to allow longer names of _objects._ Mainly, weapons and paperwork (which gets absolutely ridiculous when you can't append even 1 character to a request console paper's name).

Reply:
This will allow for mobs to have large names too(of course they can already have long ass 64 character last name).

Why do you need more than 26 characters names for objects, that's around 5-6 words. Give me an example of name that you need longer than 26 characters

Reply:
I am aware of that, as is already clear from reading my reply. I don't care.

Are you aware that all papers in request consoles are already longer than 26 characters? You cannot add anything to their name, because that just cuts the name at the 26 character mark. I can also definitely see valid use cases for names of guns or circuits (which I believe use this value as well). 

26 symbols is definitely not always 5 words, *especially* in the context of things that are named. For example, "more than 26 characters names for objects" is 41 symbols. "NSS Aurora Technology Progress Report" is 37 characters. 

You also give no arguments _against_ long names of objects aside from "well, they don't need to be that long". I disagree, I do think they may be that long.

Reply:
Alright, do you approve this @skull132 ? What is your take on this?

Reply:
I see no issue with this.